src/components/footer/footer.js

```diff
@@ -1,27 +1,61 @@
-import { Button, Container, Grid, Link, makeStyles } from '@material-ui/core';
+import { Container, Grid, Link, makeStyles, Typography } from '@material-ui/core';
 import * as React from 'react';
 import styles from './styles';
+import StatButton from './components/StatButton';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useTotalPrizeValue } from '../../features/winners/apollo/total';
 
 const useStyles = makeStyles(styles);
 
 const Footer = ({ variant }) => {
   const { t } = useTranslation();
   const classes = useStyles();
 
+  //Holders Data
+  const cadets = useSelector(state => state.holdersReducer.totalHolders || 0);
+
+  //Buybacks Data
+  const lastWeekBuybacks = useSelector(state => state.buybacksReducer.lastBuybacksTotal || 0);
+  const potsPrice = useSelector(state => state.pricesReducer.prices['POTS'] || 0);
+  const lastWeekBuybackUSD = (lastWeekBuybacks * potsPrice).toLocaleString(undefined, {
+    maximumFractionDigits: 0,
+  });
+
+  //TVL Data
+  const tvl = useSelector(state => state.vaultReducer.totalTvl || 0).toLocaleString(undefined, {
+    maximumFractionDigits: 0,
+  });
+
+  //Total Prize Data
+  const { total } = useTotalPrizeValue();
+  const totalFormatted = total.toLocaleString(undefined, {
+    maximumFractionDigits: 0,
+  });
+
   return (
     <React.Fragment>
       <Container maxWidth="xl" className={variant === 'light' ? classes.light : ''}>
         <Grid container spacing={2} className={classes.footer}>
-          <Grid item xs={12}>
-            <Button
-              href={'https://moonpot.com/alpha'}
-              className={classes.footerBtn}
-              variant={'outlined'}
-              size={'small'}
-            >
-              {t('buttons.moonpotAlpha')}
-            </Button>
+          <Grid item xs={12} className={classes.footerLinks}>
+            <Link href={'https://docs.moonpot.com/'}>
+              <Typography className={classes.textButton}>{t('header.docs')}</Typography>
+            </Link>
+            <Link href={'https://moonpot.com/alpha'}>
+              <Typography className={classes.textButton}>{t('header.articles')}</Typography>
+            </Link>
+            <Link href={'https://vote.moonpot.com/#/'}>
+              <Typography className={classes.textButton}>{t('header.vote')}</Typography>
+            </Link>
+            <Link>
```

Comment:
Missing link for Audit, also these links have a yellow underline on hover:
![image](https://user-images.githubusercontent.com/55021052/148111312-71a726df-29f3-4188-9838-a775b1ddb407.png)

Reply:
Added the link in the last commit.

Reply:
Think you added `text-decoration: none` to the paragraph inside of the anchor tag instead of the anchor tag itself.

Reply:
Resolved in last commit